build/stf-run-ci/templates/service-telemetry-framework-index-Dockerfile.j2

```diff
@@ -0,0 +1,20 @@
+# The base image is expected to contain
```

Comment:
I don't need this template any more. The index image template is in-line within the BuildConfig definition in create_catalog.yml.

TODO: remove it

Reply:
Ummm... I fixed this on the weekend and didn't push it. I will do that now :)